bokeh/settings.py

```diff
@@ -618,6 +651,17 @@ def dev(self):
     is active.
     """)
 
+    # TODO: understand this
+    validation_exceptions = PrioritizedSetting("validation_exceptions", "BOKEH_VALIDATION_EXCEPTIONS", convert=convert_validation_exceptions, default="none", help="""
+    Settings for validation error messages.
+
+    valid values are:
+
+    - ``errors`` logs errors and warnings, no exceptions (default).
+    - ``full`` logs warnings, exception on errors.
+    - ``none`` exceptions on errors or warnings.
+    """)
+
     # Non-settings methods
 
     def bokehjsdir(self) -> str:
```

Comment:
I forgot in the review: the `strict` property should be removed as part of this PR